src/csharp/Grpc.Microbenchmarks/UnaryCallOverheadBenchmark.cs

```diff
@@ -0,0 +1,93 @@
+#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using Grpc.Core;
+using Grpc.Core.Internal;
+using System;
+
+namespace Grpc.Microbenchmarks
+{
+    // this test creates a real server and client, measuring the inherent inbuilt
```

Comment:
nit: this comment probably should be reworded, since this test e.g. doesn't create a server...